### PR TITLE
feat(tutorial): complete step 2

### PR DESCRIPTION
**Pull request by snidersd**
Closes #

{{short description}}

#### Changelog

**New**

- {{new thing}}

**Changed**

- {{change thing}}

**Removed**

- {{removed thing}}

### CODE REVIEW

File: src/content/RepoPage/RepoPage.js [hunk 1]
```diff
@@ -1,7 +1,71 @@
...
+    links: 'Links',
+  },
+];
 
 const RepoPage = () => {
-  return <div>REPO PAGE</div>;
+  return (
+    <div className="bx--grid bx--grid--full-width bx--grid--no-gutter repo-page">
+      <div className="bx--row repo-page__r1">
+        <RepoTable headers={headers} rows={rows} />
```
Comment (kellychurchill) [inline src/content/RepoPage/RepoPage.js]: You're missing the grid wrapper here.

Comment (snidersd) [reply src/content/RepoPage/RepoPage.js]: @kellychurchill Not sure what to do here. My step 3 was already approved and closed. #879.

**Comment by kellychurchill** [comment]
Hi @snidersd - You need to checkout your branch for step 2 (`git checkout <branchname>`), make changes, commit and push to that same branch.

**Comment by kellychurchill** [comment]
Whoops - you're missing the repositories table. See this section of step 2: https://www.carbondesignsystem.com/tutorial/react-step-2#build-repo-page

**Comment by kellychurchill** [comment]
No updates in 90 days. Closing.

**Pull request closed without merging**